core/modules/css/python/testKvInterface.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python
 
 # LSST Data Management System
-# Copyright 2013-2014 AURA/LSST.
+# Copyright 2013-2015 AURA/LSST.
 #
```

Comment:
You are creating a database and tables here but do not destroy them. It is working now because you have "create if not exists". We will run into troubles with this approach if we ever want to change schema (because old schema is left behind), or multiple users run the same test at the same time (because db name is hardcoded). I'd recommend to create unique db name each time the test starts, and destroy it at the end of the test.